src/main/kotlin/app/model/Fact.kt

```diff
@@ -0,0 +1,41 @@
+package app.model
+
+import app.Protos
+import com.google.protobuf.InvalidProtocolBufferException
+import java.security.InvalidParameterException
+
+/**
+ * Key-value statistics for fun facts.
+ */
+data class Fact(
+    var repo: Repo = Repo(),
+    var key: String = "",
+    var value: Double = 0.0,
+    var author: Author = Author()
```

Comment:
I do think this should have an optional expiration date.

Reply:
Better to use DB timestamps for expiration calculation.

Reply:
I disagree. Timestamp could depend on what a fact is. And it's better to keep fact-specific knowledge to as few places as possible. The app knows best when a fact should expire.

Reply:
Could you give an example for facts with different expiration interval?

Reply:
Some facts can be transient. E.g. "I just used C++11 for the first time". This should expire in a month or so.

Reply:
Good example. Let's discuss it offline.